var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -104,13 +98,28 @@ def cmake_args(self):
         else:
             options.append('-DGMX_GPU:BOOL=OFF')
 
-        simd_value = self.spec.variants['simd'].value
-        if simd_value == 'auto':
-            pass
-        elif simd_value == 'none':
+        simd_features = {
+            'sse2': 'SSE2',
+            'sse4_1': 'SSE4.1',
+            'avx128fma': 'AVX_128_FMA',
+            'avx256': 'AVX_256',
+            'axv128': 'AVX2_128',
+            'avx2_256': 'AVX2_256',
+            'avx512': 'AVX_512',
+            'avx512knl': 'AVX_512_KNL',
+            'qpx': 'IBM_QPX',
```

Comment:
I think here the relevant point is if we are on a BQ/Q or not, right?

Reply:
So drop?

Reply:
Dropped.